docs/framework/wcf/feature-details/system-web-routing-integration.md

```diff
@@ -4,7 +4,7 @@ ms.date: "03/30/2017"
 ms.assetid: 31fe2a4f-5c47-4e5d-8ee1-84c524609d41
 ---
 # System.Web.Routing Integration
-When hosting a Windows Communication Foundation (WCF) service in Internet Information Service (IIS) you place a .svc file in the virtual directory. This .svc file specifies the service host factory to use as well as the class that implements the service. When making requests to the service you specify the .svc file in the URI, for example: `http://contoso.com/EmployeeServce.svc`. For programmers writing REST services this type of URI is not optimal. URIs for REST services specify a specific resource and normally do not have any extensions. The <xref:System.Web.Routing> integration feature allows you to host a WCF REST service that responds to URIs without an extension. For more information about routing see [ASP.NET Routing](https://go.microsoft.com/fwlink/?LinkId=184660).  
+When hosting a Windows Communication Foundation (WCF) service in Internet Information Service (IIS) you place a .svc file in the virtual directory. This .svc file specifies the service host factory to use as well as the class that implements the service. When making requests to the service you specify the .svc file in the URI, for example: `http://contoso.com/EmployeeServce.svc`. For programmers writing REST services this type of URI is not optimal. URIs for REST services specify a specific resource and normally do not have any extensions. The <xref:System.Web.Routing> integration feature allows you to host a WCF REST service that responds to URIs without an extension. For more information about routing see [ASP.NET Routing](https://docs.microsoft.com/previous-versions/cc668201(v=vs.140)).  
```

Comment:
Should this be changed to point to the current version from asp documentation ?

Reply:
which ones were you thinking @Youssef1313? My answer would be probably not. These are old articles so probably the .NET Framework version of those would be best. But I'd however change this to the version that has a TOC:
```suggestion
When hosting a Windows Communication Foundation (WCF) service in Internet Information Service (IIS) you place a .svc file in the virtual directory. This .svc file specifies the service host factory to use as well as the class that implements the service. When making requests to the service you specify the .svc file in the URI, for example: `http://contoso.com/EmployeeServce.svc`. For programmers writing REST services, this type of URI is not optimal. URIs for REST services specify a specific resource and normally do not have any extensions. The <xref:System.Web.Routing> integration feature allows you to host a WCF REST service that responds to URIs without an extension. For more information about routing see [ASP.NET Routing](https://docs.microsoft.com/previous-versions/aspnet/cc668201(v=vs.100)).  
```